nectarchain/user_scripts/fbrun/ComputeCalibCoefs.py

```diff
@@ -88,24 +89,29 @@ def load_wfs(self,nevents,file_name,ped_only=0,ff_only=0):
         wfs_lg_tmp=np.zeros((1855,60))

```

Comment:
Instead of hardcoding the number of pixels and samples, maybe we should rely here on https://github.com/cta-observatory/ctapipe_io_nectarcam/blob/master/ctapipe_io_nectarcam/constants.py, e.g. with something like:

```python
from ctapipe_io_nectarcam import constants
[...]
wfs_lg_tmp = np.zeros((constants.N_PIXELS, constants.N_SAMPLE))